kvmagent/kvmagent/plugins/host_plugin.py

```
@@ -2839,20 +2839,44 @@ def _collect_huawei_gpu_info(self, to, opaque=None):
             return
 
         npu_infos = []
+        npu_id_map = {}  # used to map pci address to npu id
         for npu_id in npu_ids:
             r, o, e = bash_roe(gpu.get_huawei_gpu_basic_info_cmd(npu_id))
```

Ruff (0.14.8)

`bash_roe` may be undefined, or defined from star imports

(F405)